### ISSUE TITLE
IE11 does not support version 2.0 after the upgrade?

### ISSUE THREAD

**Issue by helpceo** [antoniandre/splitpanes#64]
IE is supported at 1.45.What's the problem?

Because before method IE does not support.
solution: rewrite before().

     Object.defineProperty(child, "before", {
          configurable: true,
          enumerable: true,
          writable: true,
          value: function before() {
            var argArr = Array.prototype.slice.call(arguments),
              docFrag = document.createDocumentFragment();

            argArr.forEach(function(argItem) {
              var isNode = argItem instanceof Node;
              docFrag.appendChild(
                isNode ? argItem : document.createTextNode(String(argItem))
              );
            });

            this.parentNode.insertBefore(docFrag, this);
          }
        });

**Comment by helpceo** [comment]
error:Error in mounted hook: "TypeError: 对象不支持“before”属性或方法"

**Comment by helpceo** [comment]
solution:  Object.defineProperty(child, "before", {
              configurable: true,
              enumerable: true,
              writable: true,
              value: function before() {
                var argArr = Array.prototype.slice.call(arguments),
                  docFrag = document.createDocumentFragment();

                argArr.forEach(function(argItem) {
                  var isNode = argItem instanceof Node;
                  docFrag.appendChild(
                    isNode ? argItem : document.createTextNode(String(argItem))
                  );
                });

                this.parentNode.insertBefore(docFrag, this);
              }
            });

**Comment by I-Vargas** [comment]
@antoniandre will you be restoring support for IE11 with a version 2.1.0 or will it be dropped? The currently README.md because it says it supports IE10+

**Comment by antoniandre** [comment]
Hi guys I would like to fix this but I have no windows machine at the moment so it's quite hard to test, plus I actually don't know the error, so @helpceo can you please give an explanation about:

1. What is the error
2. What is the code to reproduce
3. What is this fix, since the given code is in the webpack generated code I have no clue what you are talking about

Cheers.

**Comment by jonathan-roy** [comment]
Hi guys!

Since [ChildNode.before](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/before) is still experimental, I guess the issue comes from the line [src/components/splitpanes/splitpanes.vue:301](https://github.com/antoniandre/splitpanes/blob/4617d17a767361463f3471c8f0fff3c3cfffec7f/src/components/splitpanes/splitpanes.vue#L301) : 
```javascript
            child.before(elm)
```

@helpceo, @I-Vargas, could you try replacing it by the following one please and tell us if this works on IE10+?

```javascript
            child.parentNode.insertBefore(elm, child)
```

**Comment by helpceo** [comment]
> @antoniandre您将使用2.1.0版本恢复对IE11的支持还是将其删除？当前的README.md，因为它说它支持IE10 +



> @antoniandre will you be restoring support for IE11 with a version 2.1.0 or will it be dropped? The currently README.md because it says it supports IE10+

I have given the solution above, because the before method IE does not support

**Comment by helpceo** [comment]
> Hi guys!
> 
> Since [ChildNode.before](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/before) is still experimental, I guess the issue comes from the line [src/components/splitpanes/splitpanes.vue:301](https://github.com/antoniandre/splitpanes/blob/4617d17a767361463f3471c8f0fff3c3cfffec7f/src/components/splitpanes/splitpanes.vue#L301) :
> 
> ```js
>             child.before(elm)
> ```
> 
> @helpceo, @I-Vargas, could you try replacing it by the following one please and tell us if this works on IE10+?
> 
> ```js
>             child.parentNode.insertBefore(elm, child)
> ```

Please refer to the solution I gave you above. I have tried your solution before

**Comment by helpceo** [comment]
@antoniandre  The package build command here is also incorrect, and hopefully the next release will improve these issues

**Comment by antoniandre** [comment]
@helpceo what's wrong with the build-bundle command? 
If you want to help please write explicit messages

**Comment by antoniandre** [comment]
Thanks for your help @I-Vargas. This issue should be solved now in version 2.1.0.

**Comment by I-Vargas** [comment]
You should close this issue and update the `README.md` with the release notes for 2.1.0 and 2.1.1.
